ruby/highlight_search_result/lib/highlight_search_result.rb

```diff
@@ -0,0 +1,9 @@
+class StringModifier
+  def search_element(search_element, incoming_string)
+    incoming_string.gsub(%r(#{search_element})i, '(' + search_element + ')')
+  end
+
+  def total_search_found(search_element, incoming_string)
+    incoming_string.split(" ").grep(%r(#{search_element})i).length
```

Comment:
Lets discuss tomorrow. We basically need to do some thinking around how to write the API for a class.

Reply:
kk sir